src/Symfony/Bundle/SecurityBundle/Command/InitAclCommand.php

```diff
@@ -93,7 +89,8 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        // BC to be removed in 4.0
+        (new SymfonyStyle($input, $output instanceof ConsoleOutputInterface ? $output->getErrorOutput() : $output))->warning('Command "init:acl" is deprecated since version 3.4 and will be removed from SecurityBundle in 4.0. Register symfony/acl-bundle and use "acl:init" instead.');
```

Comment:
`Register symfony/acl-bundle and use ...` -> `Install symfony/acl-bundle and use ...` ?